<!-- BEGIN KEY -->
public class com.sun.jdi.connect.TransportTimeoutException extends java.io.IOException
<!-- END KEY -->
<!-- BEGIN DATA -->
 This exception may be thrown as a result of a timeout
 when attaching to a target VM, or waiting to accept a
 connection from a target VM.

 <p> When attaching to a target VM, using {@link
 AttachingConnector#attach attach} this
 exception may be thrown if the connector supports a timeout
 {@link Connector.Argument connector argument}. Similiarly,
 when waiting to accept a connection from a target VM,
 using {@link ListeningConnector#accept accept} this
 exception may be thrown if the connector supports a 
 timeout connector argument when accepting.

 <p> In addition, for developers creating {@link
 com.sun.jdi.connect.spi.TransportService TransportService}
 implementations this exception is thrown when 
 {@link com.sun.jdi.connect.spi.TransportService#attach attach}
 times out when establishing a connection to a target VM,
 or {@link com.sun.jdi.connect.spi.TransportService#accept 
 accept} times out while waiting for a target VM to connect. </p>

 @see AttachingConnector#attach
 @see ListeningConnector#accept
 @see com.sun.jdi.connect.spi.TransportService#attach
 @see com.sun.jdi.connect.spi.TransportService#accept

 @since 1.5

<!-- END DATA -->
<!-- BEGIN KEY -->
public com.sun.jdi.connect.TransportTimeoutException()
<!-- END KEY -->
<!-- BEGIN DATA -->
 Constructs a <tt>TransportTimeoutException</tt> with no detail
 message.

<!-- END DATA -->
<!-- BEGIN KEY -->
public com.sun.jdi.connect.TransportTimeoutException(String message)
<!-- END KEY -->
<!-- BEGIN DATA -->
 Constructs a <tt>TransportTimeoutException</tt> with the 
 specified detail message.

 @param message the detail message pertaining to this exception.

<!-- END DATA -->
